Jenkinsfiles/web-ui-pipeline.groovy

```diff
@@ -1,20 +1,18 @@
 properties([
     [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
-    [$class: 'ParametersDefinitionProperty', parameterDefinitions:
-        [
-            [$class: 'StringParameterDefinition', defaultValue: '', description: 'Branch to test, fall-backs on $BASE_BRANCH if not found.', name: 'BRANCH'],
-            [$class: 'ChoiceParameterDefinition', choices: 'master\n9.10', description: 'The branch to fallback on when $BRANCH is not found.', name: 'BASE_BRANCH'],
-            [$class: 'StringParameterDefinition', defaultValue: 'SLAVE', description: 'Slave label to be used.', name: 'SLAVE'],
-            [$class: 'BooleanParameterDefinition', defaultValue: false,  description: 'Run npm and bower cache clean?', name: 'CLEAN'],
-            [$class: 'BooleanParameterDefinition', defaultValue: true,  description: 'Should unit tests be run on Sauce Lab (or just Chrome on the slave)?', name: 'SAUCE_LAB'],
-            [$class: 'BooleanParameterDefinition', defaultValue: true,  description: 'Should PRs be created if build is successful?', name: 'CREATE_PR'],
-            [$class: 'BooleanParameterDefinition', defaultValue: false, description: 'Should the functionnal tests be skipped?', name: 'SKIP_IT_TESTS'],
-            [$class: 'BooleanParameterDefinition', defaultValue: false, description: 'Should the element\'s tests be skipped?', name: 'SKIP_UNIT_TESTS'],
-            [$class: 'BooleanParameterDefinition', defaultValue: false, description: 'Should the metrics report be generated?', name: 'GENERATE_METRICS'],
-            [$class: 'StringParameterDefinition', defaultValue: 'chrome', description: 'The browser to use for functional tests.', name: 'BROWSER'],
-            [$class: 'StringParameterDefinition', defaultValue: '', description: 'The path to the browser binary.', name: 'BROWSER_BINARY']
-        ]
-    ],
+    parameters([
+            string(name: 'BRANCH', defaultValue: '', description: 'Branch to test, fall-backs on $BASE_BRANCH if not found.', trim: false),
+            choice(name: 'BASE_BRANCH', choices: ['master', '10.10', '9.10'], description: 'The branch to fallback on when $BRANCH is not found.'),
```

Comment:
Since this will land on a maintenance branch, I'd say theres no need to expose other base branches?
```suggestion
            choice(name: 'BASE_BRANCH', choices: ['10.10'], description: 'The branch to fallback on when $BRANCH is not found.'),
```